File: Selenium/Activities/Activity20.java
package activity;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity20 {

	public static void main(String[] args) throws InterruptedException {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/alerts");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		WebElement confirmBtn = driver.findElement(By.id("prompt"));
		confirmBtn.click();
		
		Alert alert = driver.switchTo().alert();
		
		System.out.println("Text of the alert: " + alert.getText());
		
		alert.sendKeys("Awesome!");
		
		alert.accept();
		
		String enteredText = driver.findElement(By.id("result")).getText();
		System.out.println("Result after typing and accepting the alert: " + enteredText);
		
		driver.quit();
	}

}

File: Selenium/Activities/activity_20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, timeout=5)

    driver.get("https://training-support.net/webelements/alerts")

    print("Title of the page: ", driver.title)

    prompt_btn = driver.find_element(By.ID, "prompt")
    prompt_btn.click()

    alert = wait.until(EC.alert_is_present())
    print("Text of the Alert: ", alert.text)

    alert.send_keys("Awesome!")

    alert.accept()

    entered_text = driver.find_element(By.ID, "result").text
    print("Result after typing and accepting the alert: ", entered_text)
    